docs/modules/concepts/pages/protocols/jmap.adoc

```diff
@@ -1,4 +1,29 @@
 = JSON Meta Application Protocol (JMAP)
 :navtitle: JMAP
 
-(TODO)
+== Overview
+
+https://en.wikipedia.org/wiki/Local_Mail_Transfer_Protocol[Local Mail Transfer Protocol] ("JMAP")
+is
+
+From <<RFC8620>>:
```

Comment:
blank line seems weird to me.

Reply:
It normal, 
I copied format from other file
![image](https://github.com/apache/james-project/assets/81145350/10ad1022-94d1-46d7-b801-c09a7bf4baf6)